manifests/0000_10_config-operator_01_authentication.crd.yaml

```diff
@@ -485,11 +477,7 @@ spec:
                   description: name is the metadata.name of the referenced config
                     map
                   type: string
-              required:
```

Comment:
Would expect this to be required

Reply:
@smarterclayton wouldn't block the PR on that. It fixes too many other things. Opened https://github.com/openshift/api/pull/394 and we will follow-up to repair the required here.